docs/monitor/logging.md

```diff
@@ -105,20 +105,28 @@ Starting org.apache.kyuubi.server.KyuubiServer, logging to /Users/kentyao/tmp/ky
 
 `KYUUBI_MAX_LOG_FILES` controls how many log files will be remained after a Kyuubi server reboots.
 
-#### Custom Log4j Settings
+#### Custom Log4j2 Settings
 
-Taking control of `$KYUUBI_HOME/conf/log4j.properties` will also give us the ability of customizing server logging as we want.
+Taking control of `$KYUUBI_HOME/conf/log4j2.xml` will also give us the ability of customizing server logging as we want.
 
 For example, we can disable the console appender and enable the file appender like,
 
-```properties
-log4j.rootCategory=INFO, FA
-log4j.appender.FA=org.apache.log4j.FileAppender
-log4j.appender.FA.append=false
-log4j.appender.FA.file=log/dummy.log
-log4j.appender.FA.layout=org.apache.log4j.PatternLayout
-log4j.appender.FA.layout.ConversionPattern=%d{HH:mm:ss.SSS} %t %p %c{2}: %m%n
-log4j.appender.FA.Threshold=DEBUG
+```xml
+<Configuration status="INFO">
```

Comment:
can we use content of `conf/log4j.xml.template`?

Reply:
The previous documentation described custom output to a file, so this example is retained here.